oojs/application.js

```diff
@@ -1,13 +1,18 @@
+
+
 $(document).ready(function() {
-  $('#roller button.add').on('click', function() {
-    console.log("WAT")
-    $('.dice').append('<div class="die">0</div>');
-  });
 
-  $('#roller button.roll').on('click', function() {
+  addDice = function() {
```

Comment:
You can refactor these out further into objects that have these methods, also, you can move these methods outside of the `$(document).ready()` function so it doesn't need to wait for render in order to read in the code.